textile/chat-features.textile

```diff
@@ -237,14 +262,19 @@ Each chat room can be configured individually, allowing options to be passed as
 ** @(CHA-RC2a)@ @[Testable]@ If a room is requested with invalid configuration, for example: a negative typing timeout, an @ErrorInfo@ with code @40001@ must be thrown.
-** @(CHA-RC2c)@ @[Testable]@ A room’s messages feature (@CHA-M*@) can not be disabled.
-** @(CHA-RC2b)@ @[Testable]@ If a room is configured to have a given feature disabled, then attempting to use that feature must result in an @ErrorInfo@ with code @40000@ being thrown.
-** @(CHA-RC2d)@ @[Testable]@ A room must create a lifecycle contributor for each of its enabled features and must not create a lifecycle contributor for any of its disabled features.
-** @(CHA-RC2e)@ @[Testable]@ The room lifecycle contributors have an order of precedence, which is the order in which operation sequences (e.g. attaches are applied to them). The order of precedence shall be: messages, presence, typing, reactions, occupancy.
-** @(CHA-RC2f)@ @[Testable]@ A room must only fetch a realtime channel if this channel is required by one of its lifecycle contributors.
-* @(CHA-RC3)@ Multiple features may share one realtime channel (for example, presence and messages). Each feature may have different channel modes/options that need to be applied.
-** @(CHA-RC3a)@ @[Testable]@ When retrieving a channel via @channels.get()@, features that share a channel must merge together their respective options/modes and pass the merged result to the call to @channels.get()@.
+** @(CHA-RC2c)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2b)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2d)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2e)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RC2f)@ This specification point has been removed. It was valid up until the single-channel migration.
+* @(CHA-RC3)@ Multiple chat features share one realtime channel (for example, presence and messages). Each feature may have different channel modes/options that need to be applied.
+** @(CHA-RC3a)@ @[Testable]@ When retrieving a channel via @channels.get()@, features must merge together their respective options/modes and pass the merged result to the call to @channels.get()@.
 ** @(CHA-RC3b)@ Note: SDKs shall not rely on updating the value progressively via feature spec item @RTS3c@ - as this has been soft-deprecated and may be removed in the future.
+** @(CHA-RC3c)@ @[Testable]@ Unless explicitly stated otherwise, the channel used shall be @<roomId>::$chat@. So if your roomId is @foo@, the channel will be @foo::$chat@.
+* @(CHA-RC4)@ Users may provide room options to the @rooms.get()@ call to control the behaviour of the room.
```

Comment:
I strongly feel both `CHA-RC4` and `CHA-RC4` spec points should go under `CHA-RC2`. 
Those spec points mention how room should behave when providing roomOptions.